addons/l10n_nl_edi/models/account_edi_format.py

```diff
@@ -65,13 +66,13 @@ def _get_nlcius_values(self, invoice):
     def _export_nlcius(self, invoice):
         self.ensure_one()
         # Create file content.
-        xml_content = b"<?xml version='1.0' encoding='UTF-8'?>"
+        xml_content = markupsafe.Markup("<?xml version='1.0' encoding='UTF-8'?>")
```

Comment:
is this useful? the result is stored in an attachment, and will be retrieved as b64 or bytes, so whatever Markup is generated here will be discarded immediately, no?

Reply:
The tag is found in the text of the attachment, it is not the encoding of the attachment but that of the xml of the textual content. It does not seem to me that the tag is removed.

eg (Attachments / INV_2021_05_0001_facturx.xml):
```xml
<?xml version='1.0' encoding='UTF-8'?><rsm:CrossIndustryInvoice xmlns:rsm="urn:un:unece:uncefact:data:standard:CrossIndustryInvoice:100" xmlns:ram="urn:un:unece:uncefact:data:standard:ReusableAggregateBusinessInformationEntity:100" xmlns:udt="urn:un:unece:uncefact:data:standard:UnqualifiedDataType:100">
                <rsm:ExchangedDocumentContext>
                    <ram:GuidelineSpecifiedDocumentContextParameter>
                        <ram:ID>urn:cen.eu:en16931:2017</ram:ID>
                    </ram:GuidelineSpecifiedDocumentContextParameter>
                </rsm:ExchangedDocumentContext>
                <rsm:ExchangedDocument>
                    <ram:ID>INV/2021/05/0001</ram:ID>
                    <ram:TypeCode>380</ram:TypeCode>
                    <ram:IssueDateTime>
```

Reply:
oh sorry, I was asking about the `Markup()` call on this line, vs using a simple str. If the result is directly saved into an attachment, there should be no difference between `str` and `Markup`, and it will save the `import` and simplify the code a bit, no?
(valid for each occurrence of this pattern)

Reply:
The content is not directly added, two lines down it is added to the content coming from an ir.qweb render which is also markup.